subgraph-api/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "scripts": {
+    "start": "tsx server.ts"
+  },
+  "dependencies": {
+    "@tsconfig/node20": "20.1.5",
+    "config": "3.3.12",
+    "cors": "2.8.5",
+    "dotenv": "16.4.7",
+    "express": "4.21.2",
```

Comment:
Just asking: I know express v5 has been out there in the wild for more than half a year. Any particular reason you wanted to stick with v4?

Reply:
Particularly, considering you instaled  `@types/express` - when installing `@types/*` packages, the major versions must match. So you either need `@types/express@4.x`, or bump express here

Reply:
Express devs have just marked v5 as "latest" on Monday.

I will check the types tho.

Reply:
Ah, ok! The [5.0.1](https://www.npmjs.com/package/express/v/5.0.1) appears published 6 months ago, so I guess that confused me